operations/rates.md

```diff
@@ -1,8 +1,9 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
 # Rates
 
 ## Get all rates
 
-Returns all rates \(pricing setups\) and rate groups \(condition settings\) of the default service provided by the enterprise.
+Returns all rates (pricing setups) and rate groups (condition settings) of the default service provided by the enterprise.
```

Comment:
If `RateGroups` is deprecated, we should remove rate groups from the description.

Reply:
fixed [here](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/7667e3f185fad933936815ca98a61f3a41e7a48e)